scripts/utils/mac-gpu-profiling.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * This script allows GPU Profiling on Mac using Xcode's GPU Frame Capture. Please read the instructions
+ * in the manual: https://developer.playcanvas.com/user-manual/optimization/gpu-profiling/
+ */
+var MacGPUProfiling = pc.createScript('MacGPUProfiling');
```

Comment:
Can we just commit ESM scripts from now on?

Reply:
will do at a later date, when we update the examples build process to access ESM scripts from this folder.